usr/sbin/pkcsslotd/slotmgr.c

```diff
@@ -465,12 +465,50 @@ static int config_parse_slot(const char *config_file,
     return 0;
 }
 
+static int config_parse_statistics(const char *config_file,
+                                   struct ConfigBareListNode *statistics)
+{
+    struct ConfigBaseNode *c;
+    int i;
+
+    confignode_foreach(c, statistics->value, i) {
+        DbgLog(DL3, "Config node: '%s' type: %u line: %u\n",
+               c->key, c->type, c->line);
+
+        if (c->type == CT_BARE && strcmp(c->key, "off") == 0) {
+            socketData.flags &= ~(FLAG_STATISTICS_ENABLED |
```

Comment:
That makes order somewhat important. `off, implicit` leads to different flags than `implicit, off`.  Did not check the remaining code if this might be a problem.  Personally I would not change `socketData.flags` in the off case directly, but remember to change them before returning and remove all flags at the end.

Reply:
Well, yes, but if flag FLAG_STATISTICS_ENABLED is off, then the other flags are not checked at all.  So if anywhere 'off' is specified, the FLAG_STATISTICS_ENABLED  is off and thus statistics are disabled, no matter if and how often the other keywords are specified. 

One could also specify something like `on, implicit, off, internal, on`. To be 100% correct we should also check that each keyword appears only once, and only the right combinations and order.  Not sure if it is worth to be that stricht though....